hackathon_site/dashboard/frontend/src/components/general/SideSheetRight/SideSheetRight.js

```diff
@@ -8,19 +8,19 @@ import styles from "./SideSheetRight.module.scss";
 const SideSheetRight = ({ children, title, isVisible, handleClose }) => {
     return (
         <Drawer anchor="right" open={isVisible}>
```

Comment:
Can you add `onClose={handleClose}` as a prop? That way clicking outside of the drawer will also cause it to close.